### PR TITLE
AST-683  - E2E tests site tagline

### PR DESCRIPTION
**Pull request by Swapnildhanrale**
### Description
Added test case for site tagline

### Screenshots
Options to be tested - https://share.bsf.io/p9uA4EYp

### Types of changes
Only added new test case file

### How has this been tested?
Run the command by pulling branch - `npm run test:e2e:interactive -- tests/e2e/specs/customizer/header-builder/site-identity/site-tagline.test.js`

### Checklist:
Test in autoload browser, tagline color, and font size is working or not.

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/Site-identity/site-tagline.test.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+describe( 'Site Tagline Typography settings and color settings in the customizer', () => {
+	it( 'site tagline typography and color should apply corectly', async () => {
+		const siteTagline = {
+			'display-site-tagline-responsive': {
+				// eslint-disable-next-line quote-props
+				'desktop': true,
+				// eslint-disable-next-line quote-props
+				'tablet': true,
+				// eslint-disable-next-line quote-props
+				'mobile': true,
```
Comment (Nikschavan) [inline tests/e2e/specs/customizer/header-builder/Site-identity/site-tagline.test.js:13]:
```suggestion
			'display-site-tagline-responsive': {
				'desktop': true,
				'tablet': true,
				'mobile': true,
```
Let's not disable the rules, I think this is not required anyway and this is already fixed. Can you try and remove this?

File: tests/e2e/specs/customizer/header-builder/Site-identity/site-tagline.test.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+			'body-font-variant': '800',
+			'body-font-weight': '800',
+			'body-text-transform': 'uppercase',
+			'font-size-site-tagline': {
+				// eslint-disable-next-line quote-props
+				'desktop': 22,
+				// eslint-disable-next-line quote-props
+				'tablet': 20,
+				// eslint-disable-next-line quote-props
+				'mobile': 18,
```
Comment (Nikschavan) [inline tests/e2e/specs/customizer/header-builder/Site-identity/site-tagline.test.js:25]:
```suggestion
				'desktop': 22,
				'tablet': 20,
				'mobile': 18,
```

**Review by Nikschavan — CHANGES_REQUESTED** [review]

File: tests/e2e/specs/customizer/header-builder/Site-identity/site-tagline.test.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../../utils/set-customize';
+
+describe( 'Site Tagline Typography settings and color settings in the customizer', () => {
+	it( 'site tagline typography and color should apply corectly', async () => {
+		const siteTagline = {
+			'display-site-tagline-responsive': {
+				'desktop': true,
+				'tablet': true,
+				'mobile': true,
```
Comment (Nikschavan) [inline tests/e2e/specs/customizer/header-builder/Site-identity/site-tagline.test.js:10]:
```suggestion
				desktop: true,
				tablet: true,
				mobile: true,
```

File: tests/e2e/specs/customizer/header-builder/Site-identity/site-tagline.test.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+			},
+			'body-font-family': 'Raleway, sans-serif',
+			'body-font-variant': '800',
+			'body-font-weight': '800',
+			'body-text-transform': 'uppercase',
+			'font-size-site-tagline': {
+				'desktop': 22,
+				'tablet': 20,
+				'mobile': 18,
```
Comment (Nikschavan) [inline tests/e2e/specs/customizer/header-builder/Site-identity/site-tagline.test.js:19]:
```suggestion
				desktop: 22,
				tablet: 20,
				mobile: 18,
```

**Review by Nikschavan — CHANGES_REQUESTED** [review]

**Review by shubhw12 — APPROVED** [review]

**Review by premanshup — CHANGES_REQUESTED** [review]

**Review by premanshup — APPROVED** [review]

File: tests/e2e/utils/post.js [hunk 5]
```diff
@@ -28,4 +28,15 @@ export const TPOGRAPHY_TEST_POST_CONTENT = `<!-- wp:paragraph -->
 
 <!-- wp:heading {"level":6} -->
 <h6>Heading 6</h6>
+<!-- /wp:heading -->
+
+<!-- wp:list-item {"level":7} -->
```
Comment (Nikschavan) [inline tests/e2e/utils/post.js:33]: I am getting this error @swapnilde 

Image 2021-08-05 at 11 06 48 AM


Can you confirm if this block exists on your site?

Comment (Nikschavan) [reply tests/e2e/utils/post.js:33]: @premanshup is this working for you?

Comment (premanshup) [reply tests/e2e/utils/post.js:33]: @Nikschavan _checking_

Comment (premanshup) [reply tests/e2e/utils/post.js:33]: @Nikschavan This is fixed here - https://github.com/brainstormforce/astra/pull/3080. Please review.

**Pull request merged**